docs/node_sdk_release.md

```diff
@@ -0,0 +1,36 @@
+# Node SDK Release Process Document
+
+Using v1.4 branch and last release of v1.4.15 as an example throughout.
+
+## Pre-Check
+
+Check top level `package.json` file properties **version** and **tag** are one above the current release and correct for publishing snapshots.  Expect *v1.4.16-snapshot* and *unstable-1.4*.  If this is not the case then builds are likely to fail.  Consider fixing and creating a PR or move straight onto submit a PR for a new release.
```

Comment:
The version in `package.json` should be of the form `1.4.16-snapshot`. No `v` at the start.

Reply:
Changed